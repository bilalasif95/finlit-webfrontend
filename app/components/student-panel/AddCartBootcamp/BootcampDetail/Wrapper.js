import styled from 'styled-components';

const Wrapper = styled.div`
  .addcart_detail {
    .main_img {
      margin: 0 0 1rem;
      width: 100%;
      img {
        width: 100%;
      }
    }
    .header {
      align-items: center;
      display: flex;
      justify-content: space-between;
      padding: 1rem 0;
      width: 100%;
      h4 {
        color: #484848;
        margin: 0;
      }
      .like_share {
        button {
          background-color: transparent;
          border: none;
          color: #8b8b8b;
          margin-left: 1rem;
          padding: 0;
        }
      }
    }
    .date_location {
      display: flex;
      flex-direction: column;
      justify-content: flex-start;
      width: 100%;
      .item {
        align-items: center;
        display: flex;
        font-size: 0.875rem;
        flex-direction: row;
        justify-content: flex-start;
        svg {
          color: #8b8b8b;
        }
        p {
          color: #8b8b8b;
          margin: 0 0.5rem;
        }
        .value {
          color: #000;
        }
      }
      .time-item {
        margin-top: 10px;
      }
    }
    .tabs_cont {
      margin: 30px 0 0 25px;
      width: 75%;
      .shedule-head {
        color: #484848;
        font-size: 20px;
      }
      .nav-tabs {
        border-bottom: 1px solid #e4e4e4;
        padding: 0 0 0 1rem;
        .nav-link.active {
          color: #000;
          background-color: #fff;
          border-top: none;
          border-left: none;
          border-right: none;
          border-bottom: 3px solid #ffd416;
        }
        .nav-link {
          border-top: none;
          border-left: none;
          border-right: none;
          border-bottom: 3px solid #fff;
          color: #8b8b8b;
          cursor: pointer;
          font-size: 0.875rem;
          margin-bottom: 0;
          &:hover {
            border-top: none;
            border-left: none;
            border-right: none;
            border-bottom: 3px solid #fff;
          }
        }
      }
      .tab-content > .tab-pane {
        padding: 1rem 0 0;
        .weekdays {
          display: flex;
          justify-content: space-between;
          align-items: center;
          &:not(:last-child) {
            margin-bottom: 15px;
          }
          .name {
            font-size: 14px;
            color: #b8b8b;
          }
          .time {
            font-size: #000000;
            color: #000000;
          }
        }
        h4 {
          color: #484848;
          font-size: 1.25rem;
          margin: 0;
        }
        p {
          color: #8b8b8b;
          font-size: 0.875rem;
          margin: 0;
        }
      }
    }
    .requirement {
      padding: 1.5rem;
      border: 1px solid #e4e4e4;
      border-radius: 0.313rem;
      margin-top: 30px;
      h4 {
        color: #484848;
        font-size: 1.25rem;
        line-height: 24px;
        margin-bottom: 15px;
      }
      p {
        color: #8b8b8b;
        font-size: 0.875rem;
        line-height: 24px;
      }
    }
  }
`;

export default Wrapper;
