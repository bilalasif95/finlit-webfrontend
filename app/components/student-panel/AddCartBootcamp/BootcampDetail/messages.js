/*
 * Cart Checkout Description Messages
 *
 * This contains all the text for the Cart Checkout Description component.
 */
import { defineMessages } from 'react-intl';

export const scope = 'boilerplate.components.CartCheckoutDescription';

export default defineMessages({
  Title: {
    id: `${scope}.Title`,
    defaultMessage: 'BootCamp of the printing and typesetting',
  },
  Date: {
    id: `${scope}.Date`,
    defaultMessage: 'Date:',
  },
  Time: {
    id: `${scope}.Time`,
    defaultMessage: 'Timings:',
  },
  Location: {
    id: `${scope}.Location`,
    defaultMessage: `Location:`,
  },
  RequirementText: {
    id: `${scope}.RequirementText`,
    defaultMessage: `Lorem Ipsum is simply dummy text of the printing and typesetting
    industry. Lorem Ipsum has been the industry's standard dummy text
    ever since the 1500s, when an unknown printer took a galley of type
    and scrambled it to make a type specimen book. It has survived not
    only five centuries, but also the leap into electronic typesetting,
    remaining essentially unchanged.`,
  },
  Price: {
    id: `${scope}.Price`,
    defaultMessage: `Price`,
  },
  Remove: {
    id: `${scope}.Remove`,
    defaultMessage: 'Remove',
  },
  ProvidedBy: {
    id: `${scope}.ProvidedBy`,
    defaultMessage: 'Provided By:',
  },
  ProviderName: {
    id: `${scope}.ProviderName`,
    defaultMessage: 'Joel & Natalie Rivera',
  },
  Subtotal: {
    id: `${scope}.Subtotal`,
    defaultMessage: 'Subtotal :',
  },
});
