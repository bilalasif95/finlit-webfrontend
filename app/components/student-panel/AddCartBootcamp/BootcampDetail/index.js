/*
 * Cart Checkout Description Component
 */
import React, { useState } from 'react';
import { FormattedMessage } from 'react-intl';
import { BiTimeFive, BiCalendar } from 'react-icons/bi';
import { IoMdShare } from 'react-icons/io';
import { FaRegHeart } from 'react-icons/fa';
// import { Link } from 'react-router-dom';
import { TabContent, TabPane, Nav, NavItem, NavLink, Button } from 'reactstrap';
import classnames from 'classnames';
import messages from './messages';
import Wrapper from './Wrapper';
import addcart from '../../../../images/addcart.png';

function BootcampDetail() {
  const [activeTab, setActiveTab] = useState('1');

  const toggle = tab => {
    if (activeTab !== tab) setActiveTab(tab);
  };
  return (
    <Wrapper id="list">
      <div className="addcart_detail">
        {/* <FormattedMessage {...messages.Subtotal} /> */}
        <div className="main_img">
          <img src={addcart} alt="main" />
        </div>
        <div className="header">
          <h4>
            <FormattedMessage {...messages.Title} />
          </h4>
          <div className="like_share">
            <Button>
              <FaRegHeart />
            </Button>
            <Button>
              <IoMdShare />
            </Button>
          </div>
        </div>
        <div className="date_location">
          <div className="item">
            <BiCalendar />{' '}
            <p>
              <FormattedMessage {...messages.Date} />
            </p>
            <p className="value">August 24, 2021</p>
          </div>
          <div className="item time-item">
            <BiTimeFive />{' '}
            <p>
              <FormattedMessage {...messages.Time} />
            </p>
            <p className="value">10:00 pm - 12:00 pm</p>
          </div>
        </div>
        <div className="requirement">
          <h4>About the Bootcamp</h4>
          <p>
            <FormattedMessage {...messages.RequirementText} />
          </p>
        </div>
        <div className="tabs_cont">
          <h4 className="shedule-head">Shedule</h4>
          <Nav tabs>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '1' })}
                onClick={() => {
                  toggle('1');
                }}
              >
                Monday
              </NavLink>
            </NavItem>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '2' })}
                onClick={() => {
                  toggle('2');
                }}
              >
                Tuesday
              </NavLink>
            </NavItem>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '3' })}
                onClick={() => {
                  toggle('3');
                }}
              >
                Wednesday
              </NavLink>
            </NavItem>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '4' })}
                onClick={() => {
                  toggle('4');
                }}
              >
                Thursday
              </NavLink>
            </NavItem>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '5' })}
                onClick={() => {
                  toggle('5');
                }}
              >
                Friday
              </NavLink>
            </NavItem>
            <NavItem>
              <NavLink
                className={classnames({ active: activeTab === '6' })}
                onClick={() => {
                  toggle('6');
                }}
              >
                Saturday
              </NavLink>
            </NavItem>
          </Nav>
          <TabContent activeTab={activeTab}>
            <TabPane tabId="1">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
            <TabPane tabId="2">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
            <TabPane tabId="3">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
            <TabPane tabId="4">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
            <TabPane tabId="5">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
            <TabPane tabId="6">
              <div className="weekdays">
                <p className="name">Orientation :</p>
                <p className="time">10:00 pm - 12:00 am</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture #1 :</p>
                <p className="time">12:00 pm - 01:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Lecture#2 :</p>
                <p className="time">02:00 pm - 04:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Brainstorming Excercises :</p>
                <p className="time">04:00 pm - 06:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Physical Exercises :</p>
                <p className="time">06:00 pm - 07:00 pm</p>
              </div>
              <div className="weekdays">
                <p className="name">Testing Period :</p>
                <p className="time">07:00 pm - 09:00 pm</p>
              </div>
            </TabPane>
          </TabContent>
        </div>
      </div>
    </Wrapper>
  );
}

export default BootcampDetail;
