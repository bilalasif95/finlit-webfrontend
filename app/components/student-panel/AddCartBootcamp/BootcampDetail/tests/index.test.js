import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';

import BootcampDetail from '../index';
import configureStore from '../../../../../configureStore';

describe('<BootcampDetail />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <Provider store={store}>
          <IntlProvider locale="en">
            <BrowserRouter>
              <BootcampDetail detail={{ data: {} }} />
            </BrowserRouter>
          </IntlProvider>
        </Provider>,
      )
      .toJSON();

    expect(renderedComponent).toMatchSnapshot();
  });
});
