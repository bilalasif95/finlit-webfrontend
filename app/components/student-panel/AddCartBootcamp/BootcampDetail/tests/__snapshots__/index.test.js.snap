// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BootcampDetail /> should render and match the snapshot 1`] = `
<div
  className="Wrapper-sc-19670zq-0 gJNcBd"
  id="list"
>
  <div
    className="addcart_detail"
  >
    <div
      className="main_img"
    >
      <img
        alt="main"
        src="IMAGE_MOCK"
      />
    </div>
    <div
      className="header"
    >
      <h4>
        <span>
          BootCamp of the printing and typesetting
        </span>
      </h4>
      <div
        className="like_share"
      >
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M458.4 64.3C400.6 15.7 311.3 23 256 79.3 200.7 23 111.4 15.6 53.6 64.3-21.6 127.6-10.6 230.8 43 285.5l175.4 178.7c10 10.2 23.4 15.9 37.6 15.9 14.3 0 27.6-5.6 37.6-15.8L469 285.6c53.5-54.7 64.7-157.9-10.6-221.3zm-23.6 187.5L259.4 430.5c-2.4 2.4-4.4 2.4-6.8 0L77.2 251.8c-36.5-37.2-43.9-107.6 7.3-150.7 38.9-32.7 98.9-27.8 136.5 10.5l35 35.7 35-35.7c37.8-38.5 97.8-43.2 136.5-10.6 51.1 43.1 43.5 113.9 7.3 150.8z"
            />
          </svg>
        </button>
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M383.822 344.427c-16.045 0-31.024 5.326-41.721 15.979l-152.957-88.42c1.071-5.328 2.142-9.593 2.142-14.919 0-5.328-1.071-9.593-2.142-14.919l150.826-87.35c11.762 10.653 26.741 17.041 43.852 17.041 35.295 0 64.178-28.766 64.178-63.92C448 72.767 419.117 44 383.822 44c-35.297 0-64.179 28.767-64.179 63.92 0 5.327 1.065 9.593 2.142 14.919l-150.821 87.35c-11.767-10.654-26.741-17.041-43.856-17.041-35.296 0-63.108 28.766-63.108 63.92 0 35.153 28.877 63.92 64.178 63.92 17.115 0 32.089-6.389 43.856-17.042l151.891 88.421c-1.076 4.255-2.141 8.521-2.141 13.847 0 34.094 27.806 61.787 62.037 61.787 34.229 0 62.036-27.693 62.036-61.787.001-34.094-27.805-61.787-62.035-61.787z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      className="date_location"
    >
      <div
        className="item"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7 11H9V13H7zM7 15H9V17H7zM11 11H13V13H11zM11 15H13V17H11zM15 11H17V13H15zM15 15H17V17H15z"
          />
          <path
            d="M5,22h14c1.103,0,2-0.897,2-2V8V6c0-1.103-0.897-2-2-2h-2V2h-2v2H9V2H7v2H5C3.897,4,3,4.897,3,6v2v12 C3,21.103,3.897,22,5,22z M19,8l0.001,12H5V8H19z"
          />
        </svg>
         
        <p>
          <span>
            Date:
          </span>
        </p>
        <p
          className="value"
        >
          August 24, 2021
        </p>
      </div>
      <div
        className="item time-item"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12,2C6.486,2,2,6.486,2,12s4.486,10,10,10s10-4.486,10-10S17.514,2,12,2z M12,20c-4.411,0-8-3.589-8-8s3.589-8,8-8 s8,3.589,8,8S16.411,20,12,20z"
          />
          <path
            d="M13 7L11 7 11 12.414 14.293 15.707 15.707 14.293 13 11.586z"
          />
        </svg>
         
        <p>
          <span>
            Timings:
          </span>
        </p>
        <p
          className="value"
        >
          10:00 pm - 12:00 pm
        </p>
      </div>
    </div>
    <div
      className="requirement"
    >
      <h4>
        About the Bootcamp
      </h4>
      <p>
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting
    industry. Lorem Ipsum has been the industry's standard dummy text
    ever since the 1500s, when an unknown printer took a galley of type
    and scrambled it to make a type specimen book. It has survived not
    only five centuries, but also the leap into electronic typesetting,
    remaining essentially unchanged.
        </span>
      </p>
    </div>
    <div
      className="tabs_cont"
    >
      <h4
        className="shedule-head"
      >
        Shedule
      </h4>
      <ul
        className="nav nav-tabs"
      >
        <li
          className="nav-item"
        >
          <a
            className="active nav-link"
            onClick={[Function]}
          >
            Monday
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
          >
            Tuesday
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
          >
            Wednesday
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
          >
            Thursday
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
          >
            Friday
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
          >
            Saturday
          </a>
        </li>
      </ul>
      <div
        className="tab-content"
      >
        <div
          className="tab-pane active"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
        <div
          className="tab-pane"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
        <div
          className="tab-pane"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
        <div
          className="tab-pane"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
        <div
          className="tab-pane"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
        <div
          className="tab-pane"
        >
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Orientation :
            </p>
            <p
              className="time"
            >
              10:00 pm - 12:00 am
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture #1 :
            </p>
            <p
              className="time"
            >
              12:00 pm - 01:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Lecture#2 :
            </p>
            <p
              className="time"
            >
              02:00 pm - 04:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Brainstorming Excercises :
            </p>
            <p
              className="time"
            >
              04:00 pm - 06:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Physical Exercises :
            </p>
            <p
              className="time"
            >
              06:00 pm - 07:00 pm
            </p>
          </div>
          <div
            className="weekdays"
          >
            <p
              className="name"
            >
              Testing Period :
            </p>
            <p
              className="time"
            >
              07:00 pm - 09:00 pm
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
